docs/read_the_docs/source/workflow_outputs/csv_tables/cost_multipliers.csv

```diff
@@ -7,6 +7,8 @@ upgrade_costs.floor_area_conditioned_ft_2,ft^2,"Floor Area, Conditioned"
 upgrade_costs.floor_area_conditioned_infiltration_reduction_ft_2_delta_ach_50,ft^2 * Delta ACH50,"Floor Area, Conditioned * Infiltration Reduction"
 upgrade_costs.floor_area_foundation_ft_2,ft^2,"Floor Area, Foundation"
 upgrade_costs.floor_area_lighting_ft_2,ft^2,"Floor Area, Lighting"
+upgrade_costs.flow_rate_cooling_air_cfm,cfm,"Flow Rate, Cooling Air"
+upgrade_costs.flow_rate_heating_air_cfm,cfm,"Flow Rate, Heating Air"
```

Comment:
Should we (somehow) include the design airflow rates of the upgrade building? (If they are greater than the max allowed values, then just the max values would get reported.)